src/redisraft.c

```diff
@@ -598,6 +599,70 @@ static bool handleInterceptedCommands(RedisRaftCtx *rr,
     return false;
 }
 
+static bool getDenyOomStatus(RedisRaftCtx *rr, RedisModuleCtx *ctx, RaftRedisCommandArray *cmds)
+{
+    for (int i = 0; i < cmds->len; i++) {
+        RaftRedisCommand * cmd = cmds->commands[i];
+
+        if (cmd->argc < 1) {
+            continue;
+        }
+
+        size_t cmd_len;
+        const char * cmd_str = RedisModule_StringPtrLen(cmd->argv[0], &cmd_len);
+
+        int nokey;
+        if (RedisModule_DictGetC(rr->command_deny_oom_dict, (char *) cmd_str, cmd_len, &nokey)) {
+            return true;
+        } else if (nokey) {
+            enterRedisModuleCall();
```

Comment:
I suggest moving the code under `nokey` condition that call to `INFO` and analyse the result into a separate method - `getDenyOomStatus()`, update the dict accordingly in a separate method too, and rename this method to 'isDenyOomCommands()`

Reply:
ok, restructured